modules/builds-source-secrets-entitlements.adoc

```diff
@@ -10,7 +10,7 @@ Builds that use Red Hat subscriptions to install content must include the entitl
 
 .Prerequisites
 
-You must have access to Red Hat entitlements through your subscription, and the entitlements must have separate public and private key files.
+You must have access to Red Hat entitlements through your subscription. The entitlement secret is automatically created by the insights operator.
```

Comment:
Does the docs say where the secret is created/available? If not then I would add it (it's created in the `openshift-config-managed` namespace)

Reply:
It is mentioned in this part of the docs:
https://docs.openshift.com/container-platform/4.10/support/remote_health_monitoring/insights-operator-simple-access.html

Reply:
OK thanks